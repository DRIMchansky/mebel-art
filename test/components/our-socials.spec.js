import React from 'react'
import renderer from 'react-test-renderer'
import { shallow } from 'enzyme'
import OurSocials from '../../src/components/our-socials'

describe('OurSocials', () => {
  it('renders correctly', () => {
    const tree = shallow(<OurSocials />)
    expect(tree).toMatchSnapshot()
  })
})
