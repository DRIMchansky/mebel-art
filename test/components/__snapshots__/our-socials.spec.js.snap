// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OurSocials renders correctly 1`] = `
<styled.section>
  <WithStyles(ForwardRef(Container))
    maxWidth="md"
  >
    <Title>
      <h4>
        Свяжитесь с нами как вам удобно!
      </h4>
    </Title>
    <SocialHorizontal />
  </WithStyles(ForwardRef(Container))>
</styled.section>
`;
